frictionless_ckan_mapper/ckan_to_frictionless.py

```diff
@@ -1,118 +1,123 @@
+'''
+Convert CKAN packages and resources to their
+Frictionless equivalent.
+'''
+# Standard library imports
 import json
 
+# Third-party library imports
 import slugify
 
+try:
+    json_parse_exception = json.decoder.JSONDecodeError
+except AttributeError:  # Testing against Python 2
+    json_parse_exception = ValueError
+
 
 class CKANToFrictionless:
+    '''Set of methods to convert CKAN packages and resources to their
+    Frictionless equivalent.'''
+
+    @staticmethod
+    def _resource_keys_to_remove(resource):
+        # TODO: Do we want to keep everything except `package_id` and
+        # `position`? The current test
+        # `test_resource_path_is_set_even_for_uploaded_resources`
+        # is not expecting `url_type` in its output.
+        resource_keys_to_remove = ['package_id', 'position', 'url_type']
```

Comment:
Move to class / module level so this doesn't get redefined on every call. Also, if you don't plan on modifying this list at runtime, probably best to use a tuple so it is known to be immutable.